doc/user/core.md

```diff
@@ -66,41 +66,25 @@ avoid copies.
 
 ---
 
- * `MakeAlias(a, mat, rows, cols, strict=true)`
+ * `MakeAlias(a, mat, rows, cols, offset=0, strict=true)`
    - Make `a` into an alias of `mat` with the given size.
+   - If `offset` is `0` then make an identical alias, otherwise start making an
+     alias from a specific matrix location added to `offset`.
```

Comment:
I think a user might stumble on what `offset` actually means (or what an 'identical alias' is).  Maybe this or something like it might work?

```
If `offset` is `0`, then the alias is identical: the first element of `a` is the first element of `mat`.
Otherwise, the first element of `a` is the `offset`'th element of `mat`; elements in `mat` are
ordered in a [column-major way](../matrices.md#representing-data-in-mlpack).
```

That's just an idea, maybe it could be improved further?

Reply:
This is more than good :+1: